src/TypeSpec.Extension/Emitter.Csharp/src/lib/operation.ts

```diff
@@ -199,7 +200,9 @@ export function loadOperation(
         Summary: summary,
         Deprecated: getDeprecated(program, op),
         Description: desc,
-        Accessibility: isInternal(sdkContext, op) ? "internal" : undefined,
+        Accessibility: isInternal(sdkContext, op)
```

Comment:
Can we move `internal` into `getAccess`?

Reply:
Unfortunately, we cannot do this currently. The logic for `isInternal` is a little bit different from `getAccess`. Anyway we are going to deprecate `@internal`. Will remove this after that.